server/handler/upgrade_handler.go

```diff
@@ -72,7 +79,7 @@ func (h ClusterUpgradeHandler) ServeHTTP(w http.ResponseWriter, req *http.Reques
 		WriteData(w, "success!")
 	}
 	logutil.Logger(req.Context()).Info("upgrade op success",
-		zap.String("category", "upgrading"), zap.String("op", req.FormValue("op")), zap.Bool("hasDone", hasDone))
+		zap.String("category", "upgrading"), zap.String("op", op), zap.Bool("hasDone", hasDone))
```

Comment:
Does op mean "Operation"? I think here the better is to give the full name of "op", instead of brief.